frame/staking/src/pallet/mod.rs

```diff
@@ -1507,9 +1636,10 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			validator_stash: T::AccountId,
 			era: EraIndex,
+			page: PageIndex,
```

Comment:
I considered an alternative way where we keep this call and create another `payout_stakers_by_page` to not break existing api. When page is not passed, it will always call `page=0` for payout which makes sense for atleast the old eras where the exposure is not paged (which is same as saying we have only one page).

But I am leaning towards breaking api to explicitly tell end users that rewards are now paged and for new eras if they don't call by page, they may miss rewards outside `page=0`.

One more approach could be that we always payout rewards in sequence (page 0, then 1, then 2 and so on) and so the caller has to call this function multiple times till all rewards are paid out.